src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
As we discussed at the Clawpack developers meeting yesterday and now looking at the code I think `xlimits` and `ylimits` really should be in the `PlotDataAxes` object where they usually are making it more trivial to convert a given figure to one being plotted on Google Earth.